server/service/grpc/service.go

```diff
@@ -30,12 +31,14 @@ type Service struct {
 	// Store as map[string]*grpc.ClientConn
 	// TODO: remove unavailable connection
 	conns sync.Map
+	l     *rate.Limiter
 }
 
-func NewService(opTimeout time.Duration, h Handler) *Service {
+func NewService(opTimeout time.Duration, h Handler, r int, b int) *Service {
```

Comment:
```suggestion
func NewService(opTimeout time.Duration, h Handler, config LimiterConfig) *Service {
```
Use Use a separate limiter configuration, which will make the logic clearer